atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryAtlasDbFactory.java

```diff
@@ -70,9 +71,16 @@
 
     /**
      * @deprecated see usage below. Should be configured with the {@link InMemoryAtlasDbConfig}.
+     * To be removed whenever someone removes the deprecated constructors that don't know about atlas configs...
      */
     @Deprecated
-    private static final int DEFAULT_MAX_CONCURRENT_RANGES = 64;
+    private static final int DEFAULT_MAX_CONCURRENT_RANGES = AtlasDbConstants.DEFAULT_CONCURRENT_RANGES_PER_QUERY;
```

Comment:
Feels weird that this value should change (you set DEFAULT_CONCURRENT_RANGES_PER_QUERY to 32 not 64 above)

Reply:
Actually I don't care :shrug:

Reply:
This was on purpose fixing all of the tests, they were previously random values of 4, 16, 64, and 32 range concurrency, so I just made them all just one value.